src/tlo/analysis/utils.py

```diff
@@ -1218,7 +1218,8 @@ def construct_multiindex_if_implied(df):
 
 def mix_scenarios(*dicts) -> Dict:
     """Helper function to combine a Dicts that show which parameters should be over-written.
-     * Warnings are generated if a parameter appears in more than one Dict with a different value;
+     * If a parameter appears in more than one Dict, the value in the "right-most" dict is taken, and a UserWarning
```

Comment:
I know I'm being unnecessarily pedantic, but this should really be "last added" to the dictionary, not  right-most.